src/System.ComponentModel.Annotations/tests/CustomValidationAttributeTests.cs

```diff
@@ -69,6 +69,26 @@ public static void Constructor(Type validatorType, string method)
             Assert.Equal(method, attribute.Method);
         }
 
+        [Fact]
+        public void FormatErrorMessage_NoError_ContainsName()
```

Comment:
It's not quite that validation passes. It's actually that no validation has ever been performed on that instance - i've updated the comments accordingly